package.json

```diff
@@ -9,8 +9,12 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
     "@headlessui/react": "^1.7.0",
     "@heroicons/react": "^2.0.8",
+    "@mui/icons-material": "^5.10.3",
```

Comment:
use icons from tailwind (see image for example), reason being we don't want to use all kinds of different libraries for similiar functionalities to avoid conflicts, unless there's a good reason. 
<img width="704" alt="image" src="https://user-images.githubusercontent.com/59902670/190164170-39a7fa0e-ee67-42e8-8b75-1a86dc31a352.png">

Reply:
https://heroicons.com/